cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaBrokerConfigurationBuilder.java

```diff
@@ -638,11 +690,11 @@ public KafkaBrokerConfigurationBuilder withAuthorization(String clusterName, Kaf
      * @param clusterName Name of the cluster
      * @param superUsers Super-users list who have all the rights on the cluster
      * @param authorization The authorization configuration from the Kafka CR
-     * @param useKRaft      Use KRaft mode in the configuration
+     * @param isKRaft      Use KRaft mode in the configuration
```

Comment:
We seem to use `useKRaft` in general and not `isKRaft`.

Reply:
So as related to the other comment, it's more about "is KRaft in the current configuration" (which doesn't mean it's a full KRaft cluster). I am using the `useKRaft` more as "KRaft is enabled" through the feature gate flag.

Anyway, I am not ANDing this feature gate flag with the rest of checks so maybe something is missing here.
If KRaft feature gate is disabled and you try to deploy KRaft or migration you should got an error ... it's something I have to add.

Reply:
> So as related to the other comment, it's more about "is KRaft in the current configuration" (which doesn't mean it's a full KRaft cluster).

You can also call it `useKRaft`, be consistent with what we use elsewhere and say that it is about _using KRaft in the configuration_ 😉